code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -101,9 +101,19 @@
 	return
 
 /turf/open/floor/plating/asteroid/singularity_act()
+	src.ChangeTurf(/turf/open/space)
```

Comment:
This will fuck up lavaland pretty badly

You don't need to use src like that

This is snowflake in general